ecommerce/enterprise_coupons/backends/db.py

```diff
@@ -0,0 +1,34 @@
+from ecommerce.enterprise_coupons.exceptions import CatalogueNotFoundError, CouponNotFoundError, InvalidCouponError
+from .base import CouponBackend
+from oscar.core.loading import get_model
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class DatabaseBackend(CouponBackend):
+    """Backend that reads coupons from the database."""
+    
+    def __init__(self):
+        self.Voucher = get_model('voucher', 'Voucher')
```

Comment:
Why is it necessary to set this attribute as part of the class init? I think you are just using it to obtain the model data, but you are not altering or modifying this attribute. @daviidco